gitlab/v3/objects.py

```diff
@@ -1496,6 +1496,18 @@ class ProjectFileManager(BaseManager):
     obj_cls = ProjectFile
 
 
+class ProjectPipelineSchedule(GitlabObject):
```

Comment:
I don't think pipelines schedule are implemented in the v3 API, so this file should probably not be changed.